PubNubSwiftConsole/Classes/UI/CollectionView/Cells/PublishStatusCollectionViewCell.swift

```diff
@@ -15,6 +15,9 @@ protocol PublishStatusItem: Item {
     var operation: String {get}
     var creationDate: NSDate {get}
     var statusCode: Int {get}
+    var information: String {get}
+    var timeToken: NSNumber? {get}
+    var error: AnyObject? {get}
```

Comment:
why is error of type AnyObject? And why is it optional? Isn't the PubNub status always Bool?